apps/public-docsite-v9/src/Concepts/BrowserSupportMatrix.stories.mdx

```diff
@@ -18,10 +27,78 @@ FluentUI supports browsers that are at or above the indicated versions below:
 | ------------- | ------------------ | ------- |
 | >=14.5        | >=84               | >=16    |
 
-FluentUI will ensure that no browser, EcmaScript(ES), or CSS features that are incompatible with the support matrix laid out above are introduced.
-FluentUI will not be providing polyfills for features that we expect our browser support matrix to already support. This means that consumers who require the
-use/support of older browsers that don't support certain browser, ES, or CSS features (i.e. `ResizeObserver`),
-are expected to implement their own polyfills or use a third-party one.
+FluentUI will ensure that no browser, ECMAScript, or CSS features that are incompatible with the support matrix laid out above are introduced.
+For consumers who require the use/support of older browsers, please see the partial support matrix and polyfill sections below for more information
+on what will need to be done.
+
+## Partial Browser Support Matrix:
+
+FluentUI introduces the concept of a partial browser support matrix as a means to find that balance between supporting modern browsers and older browsers.
+The partial browser matrix below will have limitations that will require consumers to transpile down and/or use polyfills to cover the support gaps. The feature gaps
```

Comment:
```suggestion
The partial browser matrix below will have limitations that require consumers to transpile down and/or use polyfills to cover the support gaps. The feature gaps
```

Repeated "will"

I know I commented earlier about saying "can", rather than "must", but in this case this seems OK to me. Perhaps because it's more of a requirement.